### PR TITLE
Adds offsets for moving the badge down or right.

### PR DESCRIPTION
**Pull request by markrickert**
With tests and readme update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,13 @@ badge.backgroundColor = [UIColor colorWithRed:49/255.0 green:69/255.0 blue:122/2
 
 ![demo](Screenshots/demo1.gif)
 
+If you want to offset the badge because it's not placed quite right:
+
+```ruby
```
Comment (mamaral) [inline README.md]: Any reason why you used ruby rather than objective-c here?

File: README.md [hunk 2]
```diff
@@ -43,6 +43,13 @@ badge.backgroundColor = [UIColor colorWithRed:49/255.0 green:69/255.0 blue:122/2
...
+badge.topOffset = 10   // Moves the badge down 10 points.
+badge.rightOffset = 10 // Moves the badge right 10 points.
```
Comment (mamaral) [inline README.md]: We'll want semi-colons here as well.

File: Source/GIBadgeView.h [hunk 3]
```diff
@@ -28,6 +28,11 @@
  */
 @property (nonatomic, strong) UIFont *font;
 
+/**
+ * The padding of the badge from the upper right
+ */
+@property (nonatomic, strong) NSNumber *topOffset;
```
Comment (mamaral) [inline Source/GIBadgeView.h]: I'd prefer it if these were `CGFloat` properties.

**Comment by mamaral** [comment]
I'm definitely a fan of this - with a few modifications addressing my comments, (and updating the tests/comments/README with the changes) I'd love to merge this in. Thanks!

File: README.md (hunk 1 above)
Comment (markrickert) [inline README.md]: lol. whoops. I write a LOT of ruby. Just force of habit.

File: README.md (hunk 2 above)
Comment (markrickert) [inline README.md]: Will fix...

**Comment by markrickert** [comment]
Everything should be fixed :)

**Comment by mamaral** [comment]
Everything looks awesome. I appreciate it! I'm at a wedding today and tomorrow, but will try to get this merged ASAP.

**Comment by markrickert** [comment]
No worries. Glad you like it!

**Pull request merged**